contracts/KYCRecordRegistry2.sol

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Based on code from MACI (https://github.com/appliedzkp/maci/blob/7f36a915244a6e8f98bacfe255f8bd44193e7919/contracts/sol/IncrementalMerkleTree.sol)
```

Comment:
I think we can remove this reference because it is not an incremental merkle tree anymore.